quickstarts/kind/index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Kubernes Kind Quickstart
+layout: default
+---
+
+Kubernetes Kind is a single-node Kubernetes cluster implemented as a single Docker image that runs as a container.
+It was primarily designed for testing Kubernetes itself, but may be used for local development or CI.
+
+When using a local install of Minikube or Minishift, the Kubernetes cluster is started inside a virtual machine, running a Linux kernel and a Docker daemon, consuming extra CPU and RAM.
+
+Kind, on the other hand, requires no additional VM - it simply runs as a linux container with a set of processes using the same Linux kernel used by your Docker daemon.
```

Comment:
I wonder how much this really applies outside of Linux ... VM is still a VM and I do not think this is really more efficient on MacOS or Windows. Should we maybe add something like _especially on Linux_?

Reply:
In my experience on MacOS it still makes a difference.
Sure, we can add 'especially on Linux'.